backend/src/main/java/gov/cdc/usds/simplereport/api/converter/FhirConverter.java

```diff
@@ -640,9 +645,21 @@ public static ServiceRequest convertToServiceRequest(
     serviceRequest.setId(id);
     serviceRequest.setIntent(ServiceRequestIntent.ORDER);
     serviceRequest.setStatus(status);
+
     if (StringUtils.isNotBlank(requestedCode)) {
       serviceRequest.getCode().addCoding().setSystem(LOINC_CODE_SYSTEM).setCode(requestedCode);
     }
+
+    serviceRequest.addExtension(
+        new Extension()
+            .setUrl(ORDER_CONTROL_EXTENSION_URL)
+            .setValue(
+                new CodeableConcept()
+                    .addCoding(
+                        new Coding()
+                            .setSystem(ORDER_CONTROL_CODE_SYSTEM)
+                            .setCode(ORDER_CONTROL_CODE_OBSERVATIONS))));
+
```

Comment:
Just a slightly cleaner way of doing this. If you do `addExtension()` that will create the new extension for you and you don't need to do new `Extension()`. Same thing for `.addCoding()`
```suggestion
    serviceRequest
        .addExtension()
        .setUrl(ORDER_CONTROL_EXTENSION_URL)
        .setValue(
            new CodeableConcept()
                .getCodingFirstRep()
                .setSystem(ORDER_CONTROL_CODE_SYSTEM)
                .setCode(ORDER_CONTROL_CODE_OBSERVATIONS));
```

Reply:
By the way I think you were wondering how you could add suggestions. In your github comments theres a little page with the +- thing that you can click on. 
![image](https://user-images.githubusercontent.com/10108172/228263099-dea57bad-0eef-4477-a274-5f10813db854.png)

Reply:
I cannot pinpoint the reason but when I try to set the coding with the more direct way (even with the suggestions above) and I don't use the `addCoding + new Coding` combo the structure does not get generated a CodeableConcept  as the value of the extension like RS is expecting it.

I tried to do the clean way yesterday and spent some time debugging... I only got the right structure (CodeableConcept as the value of the extension) when setting the coding with  `new CodeableConcept().addCoding(new Coding().setSystem(ORDER_CONTROL_CODE_SYSTEM).setCode(ORDER_CONTROL_CODE_OBSERVATIONS))`

Do you have any thoughts why this could be happening?